### PR TITLE
Update documentation

### LINKED ISSUE #498: [Gesso 5] Update documentation and configuration files with updated info
- Update theme name in package and other configuration files (e.g., Gesso, not Gesso5)
- Ensure ignore files are ignoring proper files/paths, including updated image paths (.gitignore, .dockerignore, etc.)
- Update all README files with accurate and/or missing info, if it hasn't been done already (e.g., Drush commands documentation, README file in images directory related to sprite generation, info from old JS scripts README file, etc.)

### PR DESCRIPTION
**Pull request by dcmouyard**
This PR fixes #498.

i didn’t copy over the README.md that used to be in the js directory since I’m not sure how applicable it still is.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,139 +1,466 @@
...
+Sass can be compiled as part of the global styles.css file or to individual CSS
+files for use in a Drupal library.
 
-All Sass files that are compiled to individual CSS files must have a unique name,
-even if they are in different directories.
+`@use` is used to import Sass variables, mixins, and/or functions into
+individual SCSS files. [`@import` is discouraged by the Sass team and will
+eventually be phased out.](https://sass-lang.com/documentation/at-rules/import).
+This means that most files will start with
+`@use 00-config/functions/gesso as *`. This allows you to use the design token
```
Comment (kmonahan) [inline README.md]: Corey came up with a better approach, so most files will start with `@use '00-config' as *;` instead.

File: README.md [hunk 2]
```diff
@@ -1,139 +1,466 @@
...
+### Global styles
+
+Prefix the name of your Sass file with `_`, e.g. `_card.scss`. Add it to the
+appropriate aggregate file (i.e. `_components.scss`).
+
+### Individual component/library styles
+
+DO NOT prefix the name of your Sass file with `_`, e.g. `menu.scss`. Import the
+config and global aggregate files. Import your SCSS file at the top of your
+Storybook file. See `menu.stories.jsx` for an example. Don’t forget to add it to
```
Comment (kmonahan) [inline README.md]: Now that we broke out the menus, this should be `dropdown-menu.stories.jsx` (or any of the other menu examples)

File: README.md [hunk 3]
```diff
@@ -1,139 +1,466 @@
...
+break the build, so if you have a valid reason to break Stylelint rules you can
+have it ignore code in two ways:
+
+1.  Add `// stylelint-disable-next-line` to the line just before where the
+    Stylelint warning is triggered.
+
+2.  To ignore several lines, add `// stylelint-disable` before the code in
+    question and add `// stylelint-enable` afterwards.
+
+The Stylelint rules can be changed in the `.stylelintrc.yml` file. By default,
```
Comment (kmonahan) [inline README.md:171]: Change from our previous documentation, but I wonder if we should include something here that you should also leave a comment as to what the valid reason was that required you to disable stylelint (or eslint etc.) That both forces you to think about whether you really have a valid reason and can alert your code reviewer/future developers as to whether that can/should be changed. (I've had a code reviewer catch that my reason for disabling wasn't actually correct and I didn't need to disable the linter; I needed to fix the bug three lines up.)

File: README.md [hunk 4]
```diff
@@ -1,139 +1,466 @@
...
 
-### Individual Component/Library Scripts
+Prefix the name of your JavaScript file with `_`, e.g. `_Menu.es6.js`. Import it
+to the appropriate JavaScript file(s), (i.e. `primary-menu.es6.js`).
+
+### Individual component/library scripts
+
 DO NOT prefix the name of your JS file with `_`. Import your JS file at the top
-of your Storybook file. See `menu.stories.jsx` for an example. Don't forget to
+of your Storybook file. See `menu.stories.jsx` for an example. Don’t forget to
```
Comment (kmonahan) [inline README.md]: Same as with CSS, `dropdown-menu.stories.jsx` is now a better example.

File: README.md [hunk 5]
```diff
@@ -1,139 +1,466 @@
...
 add it to the `gesso.libraries.yml` file as well.
 
 ### Linting
-ESLint and Prettier are used to lint JavaScript files. ESLint uses the [Airbnb
-standards](https://github.com/airbnb/javascript/), which are [followed by Drupal](https://www.drupal.org/docs/develop/standards/javascript/javascript-coding-standards) as well.
+
+ESLint and Prettier are used to lint JavaScript files. If you have a valid
+reason to break one of the rules, you can ignore a specific line using any of
+the options in the [ESLint
+documentation](https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments).
```
Comment (kmonahan) [inline README.md:208]: If you do take the suggestion to encourage devs to write a comment with the reason, it's probably worth repeating here.

**Review by kmonahan — COMMENTED** [review]
Nothing merge blocking here, just some suggested updates of things that never got updated in the `gesso_next` README as Gesso 5 evolved.

File: README.md (hunk 1 above)
Comment (dcmouyard) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (dcmouyard) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (dcmouyard) [reply README.md:171]: Oooh, I like this! Adding more info now.

File: README.md (hunk 5 above)
Comment (dcmouyard) [reply README.md:208]: Done

File: README.md (hunk 4 above)
Comment (dcmouyard) [reply README.md]: Done

**Review by kmonahan — APPROVED** [review]

**Pull request merged**